doc/CHANGES.txt

```diff
@@ -9,17 +9,37 @@ removed.  Use --error-test-filename and --regression-test-filename to set
 the file names; old command-line argument --junit-classname has been
 removed.
 
-New command-line argument --no-npe-contract disables the "No null pointer
-exceptions without null" heuristic contract.
-
-You can control which assertions appear in regression tests with new
-command-line arguments --no-regression-assertions and
---regression-assertions-about-exceptions; old command-line argument
+You can now indicate whether you want to view exceptions as expected, error or 
+invalid behavior. The new command-line arguments --checked-exception, 
+--unchecked-exception, --npe-on-null-input, --oom-exception all take values
+of EXPECTED, ERROR, or INVALID. Expected behaviors are checked in regression
+tests, error behaviors in error-revealing tests, and any tests with invalid 
+behaviors are thrown out. (Fixes #69 and #72)
+
+You can also control whether assertions appear in regression tests with the new
+command-line argument --no-regression-assertions; old command-line argument
 --check-regression-behavior has been removed.
 
-Old command-line argument --forbid-null has been removed; use
---null-ratio=0 instead.
+The command line --test-classes has been changed to --include-only-classes to
+distance meaning from --testclass argument. (Addresses #66)
+
+Fixes a bug in which certain inherited public methods were being excluded from 
```

Comment:
This sentence has no subject, making it hord to read.
Also, I would organize the changelog from most important and user-visible changes, to bug fixes, to non-user-visible changes (if they even appear).  A bug fix should appear after changes to command-line options.